.github/workflows/run-tests.yaml

```diff
@@ -0,0 +1,31 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Test Suite for elasticsearch-operator
+
+on: [pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9]
```

Comment:
Are you sure, 3.5, 3.6 and 3.7 can be excluded ?

Reply:
Why would they be needed. The code does not run on those versions